apiserver/Makefile

```diff
@@ -120,7 +120,11 @@ OPERATOR_IMAGE_TAG ?= latest
 SECURITY_IMAGE_TAG ?= latest
 .PHONY: cluster
 cluster: kind ## Start kind development cluster.
-	$(KIND) create cluster -n $(KIND_CLUSTER_NAME) --config $(KIND_CONFIG)
+	@if kind get clusters | grep -q "^$(KIND_CLUSTER_NAME)$$"; then \
```

Comment:
@troychiu it's somewhat related to the issue: https://github.com/ray-project/kuberay/issues/3368